cartography/intel/aws/ec2/network_interfaces.py

```diff
@@ -188,56 +188,57 @@ def load_network_interface_elbv2_relations(
 
 
 @timeit
-def load_network_interface_instance_to_subnet_relations(neo4j_session: neo4j.Session, update_tag: int) -> None:
+def load_network_interface_instance_to_subnet_relations(neo4j_session: neo4j.Session, aws_account_id, update_tag: int) -> None:
     """
     Creates (:EC2Instance)-[:PART_OF_SUBNET]->(:EC2Subnet) if
     (:EC2Instance)--(:NetworkInterface)--(:EC2Subnet).
     """
     ingest_network_interface_instance_relations = """
-    MATCH (i:EC2Instance)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
+    MATCH(aa: AWSAccount{id: $AWS_ACCOUNT_ID}), (i:EC2Instance)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
     MERGE (i)-[r:PART_OF_SUBNET]->(s)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = $update_tag
     """
     logger.debug("-> Instance to subnet")
     neo4j_session.run(
-        ingest_network_interface_instance_relations, update_tag=update_tag,
+        ingest_network_interface_instance_relations, AWS_ACCOUNT_ID=aws_account_id, update_tag=update_tag,
     )
 
 
 @timeit
-def load_network_interface_load_balancer_relations(neo4j_session: neo4j.Session, update_tag: int) -> None:
+def load_network_interface_load_balancer_relations(neo4j_session: neo4j.Session, aws_account_id, update_tag: int) -> None:
     """
     Creates (:LoadBalancer)-[:PART_OF_SUBNET]->(:EC2Subnet) if
     (:LoadBalancer)--(:NetworkInterface)--(:EC2Subnet).
     """
     ingest_network_interface_loadbalancer_relations = """
-    MATCH (i:LoadBalancer)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
+    
+    MATCH(aa: AWSAccount{id: $AWS_ACCOUNT_ID}),(i:LoadBalancer)-[:NETWORK_INTERFACE]-(interface:NetworkInterface)-[:PART_OF_SUBNET]-(s:EC2Subnet)
```

Comment:
same as above, relationship between AWSAccount and LoadBalancer should be used for this query